### PR TITLE
Fix Sprite.getLocalBounds corrupting bounds

### PR DESCRIPTION
**Pull request by dev7355608**
##### Description of change

Currently `getLocalBounds` modifies and corrupts the `_bounds` of the sprite so that subsequent `getBounds(true)` calls return the local bounds instead of the global bounds. This PR fixes that.

##### Pre-Merge Checklist

- [ ] Tests and/or benchmarks are included
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

**Comment by bigtimebuddy** [comment]
Can you provide a test or reproduction of the underlying issue?

**Comment by dev7355608** [comment]
> Can you provide a test or reproduction of the underlying issue?

https://www.pixiplayground.com/#/edit/rBsga2X9LBZK4DnYr6ejx

File: packages/sprite/src/Sprite.ts [hunk 1]
```diff
@@ -417,10 +417,15 @@ export class Sprite extends Container
...
         if (this.children.length === 0)
         {
-            this._bounds.minX = this._texture.orig.width * -this._anchor._x;
-            this._bounds.minY = this._texture.orig.height * -this._anchor._y;
-            this._bounds.maxX = this._texture.orig.width * (1 - this._anchor._x);
-            this._bounds.maxY = this._texture.orig.height * (1 - this._anchor._y);
+            if (!this._localBounds)
+            {
+                this._localBounds = new Bounds();
+            }
```
Comment (bigtimebuddy) [inline packages/sprite/src/Sprite.ts:423]: Minor suggestion.
```suggestion
            this._localBounds = this._localBounds || new Bounds();
```

**Comment by bigtimebuddy** [comment]
Great, thanks for the example. We need a unit-test that handles that example for this PR. But other than that, I'm good on this fix.

**Review by GoodBoyDigital — APPROVED** [review]
nice catch @dev7355608 ! If you could add a test, so we don't regress this in the future that would be awesome. Thanks!

**Review by bigtimebuddy — APPROVED** [review]

**Comment by bigtimebuddy** [comment]
Thank you @dev7355608

**Pull request merged**